p/themes/Mapco/mapco.rtl.css

```diff
@@ -1385,3 +1373,7 @@ a, button.as-link {
 	outline: none;
 	color: #36c;
 }
+
+.icon {
+	height: inherit;
+}
```

Comment:
I guess you forgot to commit mapco.scss, that inserts this lines into the CSS file.

I would suggest to do another PR for changing the width/height behavior of `.icon` and its `height`. Let's do not do it with this PR. I can support you with a PR.